pkg/arvo/app/invite-store.hoon

```diff
@@ -0,0 +1,186 @@
+/+  *invite-json
+|%
++$  move  [bone card]
+::
++$  card
+  $%  [%diff diff]
+      [%quit ~]
+  ==
+::
++$  state
+  $%  [%0 state-zero]
+  ==
+::
++$  state-zero
+  $:  =invites
+  ==
+::
++$  diff
+  $%  [%invite-initial invites]
+      [%invite-update invite-update]
+  ==
```

Comment:
Why isn't this definition in the `/sur` file in its entirety?

(I'd personally also just add `%initial` to `invite-update`, instead of having an entirely separate diff tag for it.)

Reply:
It's specific to this app.

Reply:
Sure, this app is the only one _sending_ the diff, but any subscriber can _receive_ the diff, in case it will still want the type.

If it's intended do only send to other invite-store instances, then: 1) that breaks store rules, and 2) it's not currently happening, so might as well take it out.

Reply:
Idk man, none of the other stores have that convention. I prefer pokes and diffs to be defined in the specific file.

Reply:
@Fang-  is right, pokes and diffs are public interfaces.

Reply:
Okay. Will change.

Reply:
Have changed this now.